src/SFA.DAS.EmployerIncentives.Commands/Services/BusinessCentralApi/BusinessCentralFinancePaymentsService.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using SFA.DAS.EmployerIncentives.Abstractions.DTOs.Queries.ApprenticeshipIncentives;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.Exceptions;
+using SFA.DAS.EmployerIncentives.Enums;
+
+namespace SFA.DAS.EmployerIncentives.Commands.Services.BusinessCentralApi
+{
+    public class BusinessCentralFinancePaymentsService : IBusinessCentralFinancePaymentsService
+    {
+        private readonly HttpClient _client;
+        private string _apiVersion;
+        private int _paymentRequestsLimit;
+
+        public BusinessCentralFinancePaymentsService(HttpClient client, int paymentRequestsLimit, string apiVersion)
+        {
+            _client = client;
+            _apiVersion = apiVersion ?? "2020-10-01";
+            _paymentRequestsLimit = paymentRequestsLimit <= 0 ? 1000 : paymentRequestsLimit;
+        }
+
+        public async Task<PaymentsSuccessfullySent> SendPaymentRequestsForLegalEntity(List<PaymentDto> payments)
+        {
+            var paymentsToSend = payments.Take(_paymentRequestsLimit).ToList();
+            var content = CreateJsonContent(paymentsToSend);
+            var response = await _client.PostAsync($"payments/requests?api-version={_apiVersion}", content);
```

Comment:
Is this call idempotent?

Reply:
I believe it will accept the same payment requests (as long as they are formatted correctly) and return a 202. However, I understand if the payment request (for a given id) has already been processed it will simply skip over the new details